qa/sc_key_rotation_across_fork.py

```diff
@@ -0,0 +1,378 @@
+#!/usr/bin/env python3
+import time
+
+from httpCalls.node.getCertificateSigners import http_get_certificate_signers
+from httpCalls.node.getKeyRotationProof import http_get_key_rotation_proof
+from httpCalls.node.signSchnorrPublicKey import http_sign_schnorr_publicKey
+
+from SidechainTestFramework.sc_boostrap_info import SCNodeConfiguration, SCCreationInfo, MCConnectionInfo, \
+    SCNetworkConfiguration, SC_CREATION_VERSION_1, SC_CREATION_VERSION_2, KEY_ROTATION_CIRCUIT
+from SidechainTestFramework.sc_forging_util import *
+from SidechainTestFramework.sc_test_framework import SidechainTestFramework
+from SidechainTestFramework.scutil import bootstrap_sidechain_nodes, \
+    start_sc_nodes, generate_next_blocks, generate_next_block, generate_cert_signer_secrets
+from httpCalls.block.findBlockByID import http_block_findById
+from httpCalls.transaction.createKeyRotationTransaction import http_create_key_rotation_transaction
+from httpCalls.transaction.sendCoinsToAddress import sendCointsToMultipleAddress
+from httpCalls.wallet.createPrivateKey25519 import http_wallet_createPrivateKey25519
+from test_framework.util import assert_equal, assert_true, start_nodes, \
+    websocket_port_by_mc_node_index, forward_transfer_to_sidechain
+
+"""
+Configuration:
+    Start 1 MC node and 1 SC node.
+    SC node 1 connected to the MC node 1.
+Test:
+
+"""
+
+def convertSecretToPrivateKey(secret):
+    return secret[2:66]
+
+
+class SCKeyRotationTest(SidechainTestFramework):
```

Comment:
Check this file, seems to be a wrongly committed.
We now use only `sc_cert_key_rotation_across_epoch.py`

Reply:
Removed file